packages/ui/src/components/wui-icon-box/index.ts

```diff
@@ -0,0 +1,55 @@
+import { html, LitElement } from 'lit'
```

Comment:
wui-icon-box should be a "composite" as it uses icon component. (we can probably get rid of example composite as well).